rootconf/default/bin/onie-support

```diff
@@ -58,6 +58,11 @@ export > $save_dir/runtime-export-env.txt
 set > $save_dir/runtime-set-env.txt
 ps w > $save_dir/runtime-process.txt
 dmesg > $save_dir/dmesg.txt
+onie-syseeprom > $save_dir/onie-syseeprom.txt
+onie-sysinfo > $save_dir/onie-sysinfo.txt
+cp /etc/lsb-release > $save_dir/lsb-release.txt
+blkid > $save_dir/blkid.txt
+sgdisk -p /dev/sda > $save_dir/sgdisk.txt
```

Comment:
This line will only work for ONIE devices that have /dev/sda.  We need to make this line applicable to all ONIE machines.  Take a look at https://github.com/opencomputeproject/onie/blob/master/contrib/oce/test-nos.sh#L99 .  We'll have to do something similar here to get all machines.